scripts/copy_test_files.js

```diff
@@ -2,12 +2,128 @@
 
 const fs = require("fs-extra");
 const { join } = require("path");
+const { spawn } = require("child_process");
+
+function promiseFromChildProcess(childProcess) {
+    return new Promise(function (resolve, reject) {
+        childProcess.on("error", function (error) {
+            reject(
+                new Error(
+                    childProcess.spawnargs.join(" ") + " : " + error.message
+                )
+            );
+        });
+        childProcess.on("exit", function (code) {
+            if (code !== 0) {
+                reject(
+                    new Error(
+                        childProcess.spawnargs.join(" ") +
+                            " : exited with code " +
+                            code
+                    )
+                );
+            } else {
+                resolve();
+            }
+        });
+    });
+}
+
+const isWindows = process.platform === "win32";
+const npmCommand = isWindows ? "npm.cmd" : "npm";
+
+function ensureNpmVersion() {
+    return Promise.resolve().then(() => {
+        const npmProc = spawn(npmCommand, ["--version"], {
+            stdio: ["ignore", "pipe", "inherit"],
+        });
+        let npmVersion = "";
+        npmProc.stdout.on("data", (data) => {
+            npmVersion += data;
+        });
+        return promiseFromChildProcess(npmProc).then(() => {
+            npmVersion = npmVersion.trim();
+            let firstDot = npmVersion.indexOf(".");
+            const npmMajorVer = parseInt(
+                npmVersion.slice(0, npmVersion.indexOf("."))
+            );
+            if (npmMajorVer < 7) {
+                throw new Error(
+                    "npm version must be at least 7, version installed is " +
```

Comment:
Installing the monorepo's deps (and gluing the inter-package deps together) requires either lerna, or yarn, or npm>7.  I opted for the npm approach as I figured it's the most "vanilla" despite having been around for the least amount of time. This could run in to issues under some of your CI configs however so let's see if anything breaks and decide what to do.